justfile

```diff
@@ -38,11 +41,17 @@ sysext dir=`echo $(pwd)/cosmic-sysext` version=("nightly-" + `git rev-parse --sh
     {{ make }} -C cosmic-osd install DESTDIR={{dir}} prefix=/usr
     {{ make }} -C cosmic-panel install DESTDIR={{dir}} prefix=/usr
     {{ make }} -C cosmic-settings-daemon install DESTDIR={{dir}} prefix=/usr
-    {{ just }} rootdir={{dir}} cosmic-session/install
     {{ just }} _meson_install iced-workspaces-applet {{dir}}
     {{ just }} _meson_install user-color-editor {{dir}}
     {{ make }} -C xdg-desktop-portal-cosmic install DESTDIR={{dir}} prefix=/usr
-
+    if test {{x}} = 1; then
+        mkdir -p $HOME/.local/share/systemd/user/
+        install -Dm0644 data/wayland-proxy-virtwl.service $HOME/.local/share/systemd/user/wayland-proxy-virtwl.service
```

Comment:
Lets install to `/usr/lib/systemd/user` instead.

Reply:
it seems /usr/lib does not work yet in ubuntu